libstuff/libstuff.cpp

```diff
@@ -654,6 +654,16 @@ list<int64_t> SParseIntegerList(const string& value, char separator) {
     return valueList;
 }
 
+// --------------------------------------------------------------------------
+set<int64_t> SParseIntegerSet(const string& value, char separator) {
+    set<int64_t> valueSet;
+    list<string> strings = SParseList(value, separator);
+    for (string str : strings) {
```

Comment:
Oops, at least use `const string & str` here, it will avoid making an extra copy of each string.

Reply:
Fixed!